src/tests/multihost/basic/conftest.py

```diff
@@ -25,23 +27,39 @@ def pytest_namespace():
 
 
 @pytest.fixture(scope="class")
-def multihost(session_multihost, request):
+def multihost(session_multihost):
+    """ multihost fixture """
     return session_multihost
 
 
 @pytest.fixture(scope="session")
-def run_authselect(session_multihost, request):
+def package_install(session_multihost):
+    """ Install required packages """
+    distro = session_multihost.master[0].distro
+    pkg_list = 'authselect nss-tools 389-ds-base krb5-server'\
```

Comment:
You need to add a space after krb5-server, otherwise dnf will try to install "krb5-serveropenldap-clients".